src/appkit.cs

```diff
@@ -19698,32 +20105,32 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 		void SetAnchorAttribute (NSLayoutAttribute layoutAttribute, NSLayoutConstraintOrientation forOrientation);
 
 		[Lion, Export ("visualizeConstraints:")]
-		void VisualizeConstraints (NSLayoutConstraint [] constraints);
+		void VisualizeConstraints ([NullAllowed] NSLayoutConstraint [] constraints);
 
-        [Lion, Export ("convertRectToScreen:")]
-        CGRect ConvertRectToScreen (CGRect aRect);
+		[Lion, Export ("convertRectToScreen:")]
```

Comment:
+1 on fixing it, not really too much noise in this PR since Github makes it really easy to spot